packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -20,62 +22,79 @@ module.exports = {
   'Displays ajax error message in default': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .assert.containsText('div', defaultTranslations['Terra.ajax.error']);
   },
 
   'Displays ajax error message in de': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="de"]')
+      .pause(pauseInms)
       .assert.containsText('div', deTransaltions['Terra.ajax.error']);
   },
 
   'Displays ajax error message in en': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="en"]')
+      .pause(pauseInms)
       .assert.containsText('div', enTransaltions['Terra.ajax.error']);
   },
 
   'Displays ajax error message in en-GB': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="en-GB"]')
+      .pause(pauseInms)
       .assert.containsText('div', enGBTransaltions['Terra.ajax.error']);
   },
 
   'Displays ajax error message in en-US': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="en-US"]')
+      .pause(pauseInms)
       .assert.containsText('div', enUSTransaltions['Terra.ajax.error']);
   },
 
   'Displays ajax error message in es': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="es"]')
+      .pause(pauseInms)
       .assert.containsText('div', esTransaltions['Terra.ajax.error']);
   },
 
   'Displays ajax error message in fi-FI': (browser) => {
     browser
       .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/i18n-tests/default`)
+      .waitForElementVisible('select', 1000)
       .click('select option[value="fi-FI"]')
+      .pause(pauseInms)
       .assert.containsText('div', fiFITransaltions['Terra.ajax.error']);
```

Comment:
Nice catch. That is a typo by copy and paste. fix it in [commit](https://github.com/cerner/terra-core/pull/158/commits/2934eb8dddbc14562edb7a6698eb7707ad94a8a4)